io/FV3GFS_io.F90

```diff
@@ -703,17 +704,24 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain)
         do ix = 1, Atm_block%blksz(nb)
           i = Atm_block%index(nb)%ii(ix) - isc + 1
           j = Atm_block%index(nb)%jj(ix) - jsc + 1
-          !--- assign hprime(1:10) and hprime(15:24) with new oro stat data
-          Sfcprop(nb)%hprime(ix,1)  = oro_ls_var(i,j,1)
-          Sfcprop(nb)%hprime(ix,2)  = oro_ls_var(i,j,2)
-          Sfcprop(nb)%hprime(ix,3)  = oro_ls_var(i,j,3)
-          Sfcprop(nb)%hprime(ix,4)  = oro_ls_var(i,j,4)
-          Sfcprop(nb)%hprime(ix,5)  = oro_ls_var(i,j,5)
-          Sfcprop(nb)%hprime(ix,6)  = oro_ls_var(i,j,6)
-          Sfcprop(nb)%hprime(ix,7)  = oro_ls_var(i,j,7)
-          Sfcprop(nb)%hprime(ix,8)  = oro_ls_var(i,j,8)
-          Sfcprop(nb)%hprime(ix,9)  = oro_ls_var(i,j,9)
-          Sfcprop(nb)%hprime(ix,10)  = oro_ls_var(i,j,10)
+          ! Replace hprime(1:10) with GSL oro stat data only when using GSL
+          ! drag suite with large scale GWD and blocking as part of unified drag
+          ! suite. Otherwise, original oro stat data is used.
+          if ( (Model%gwd_opt==3 .or. Model%gwd_opt==33) .or.    &
```

Comment:
What about Model%gwd_opt==1? So it seems previous code will update hprime for all the gwd options, with the if statement, hprime will only be updated for options 3,33 or certain conditions with gsl drag.

Reply:
`gwd_opt = 1` refers to the original GFS GWD (in `gwdps.f`). For this, the separate large scale and small scale orography files are not in use. These were created specifically for the unified gravity wave drag, but they are only used to replace the default orography (that is read previously) under certain conditions. For gwd_opt = 2 or 22 (the "new" unified gravity wave drag from this PR, with (22) or without (2) extended diagnostics turned on), the new large scale orography files are only used if the flag for "large-scale GWD and blocking only" = do_gsl_drag_ls_bl is on. They are always used for gwd_opt 3 or 33 (the GSL drag suite). @mdtoyNOAA did I capture this correctly?

Reply:
Dom.  Yes, this is correct.